cmd/podman/runlabel.go

```diff
@@ -53,7 +53,6 @@ func init() {
 	flags.MarkHidden("opt2")
 	flags.MarkHidden("opt3")
 
-	flags.BoolVarP(&runlabelCommand.Pull, "pull", "p", false, "Pull the image if it does not exist locally prior to executing the label contents")
```

Comment:
We're supposed to be stable now - we can't remove command line arguments outright.

Mark hidden and print a warning if it's used.